api/datastore/mysql/mysql.go

```diff
@@ -511,7 +505,7 @@ func (ds *MySQLDatastore) GetRoutesByApp(ctx context.Context, appName string, fi
 		var route models.Route
 		err := scanRoute(rows, &route)
 		if err != nil {
-			continue
+			fmt.Println("failed to get routes by app:", err)
```

Comment:
I think these still need to continue also, so the errored route is not appended.